iina/AppDelegate.swift

```diff
@@ -198,6 +198,9 @@ class AppDelegate: NSObject, NSApplicationDelegate, SPUUpdaterDelegate {
     // register for url event
     NSAppleEventManager.shared().setEventHandler(self, andSelector: #selector(self.handleURLEvent(event:withReplyEvent:)), forEventClass: AEEventClass(kInternetEventClass), andEventID: AEEventID(kAEGetURL))
 
+    // Check for legacy pref entries and migrate them to their modern equivalents
```

Comment:
I understand the need to use "Setting" for user-facing text, but I think it's better to continue using "Preference" internally. Mainly to keep consistency, because Preference is already being used to refer to UserDefaults entries. But I also personally like using it because "Setting" is such an overloaded term that it's not immediately clear what it's referring to without more context. A web developer could refer to cookies, local storage, or server state as "settings". And often people use the word "setting" to refer to the _UI of the setting_ instead, like with the Settings sidebar.

Reply:
Sounds fine. I was rather dismayed by Apple changing this term. Imagine how much programmer time was spent on this little tweak when you add up all the time to change this in all the affected apps.